README.md

```diff
@@ -12,6 +12,12 @@ make
 sudo make install
 ```
 
+## Build Lua wrapper ##
+```bash
+./configure --enable-lua=yes
```

Comment:
@jacobkeeler `enable-lua` is not quite descriptive for this functionality. may be something like `with-lua-wrapper`